src/apps/beefy/positions.test.ts

```diff
@@ -1,5 +1,100 @@
-import { BaseBeefyVault } from './api'
-import { getDailyYieldRatePercentage } from './positions'
+import { TFunction } from 'i18next'
+import {
+  BaseBeefyVault,
+  getApyBreakdown,
+  getBeefyPrices,
+  getBeefyVaults,
+  getTvls,
+} from './api'
+import hook, { getDailyYieldRatePercentage } from './positions'
+import { NetworkId } from '../../types/networkId'
+import { Address } from 'viem'
+
+const mockT = ((x: string) => x) as TFunction
+
+const mockReadContract = jest.fn()
+jest.mock('../../runtime/client', () => ({
+  getClient: jest.fn(() => ({
+    readContract: mockReadContract,
+  })),
+}))
+
+jest.mock('./api.ts')
+
+const mockBeefyVault1: BaseBeefyVault = {
+  id: 'vault1',
+  name: 'Vault 1',
+  type: 'auto',
+  subType: 'concentrated',
+  token: 'vault1Token',
+  tokenAddress: '0x123456789' as Address,
+  tokenDecimals: 8,
+  tokenProviderId: '',
```

Comment:
These empty fields are not used by the implementation for Earn positions. I can add dummy values if wanted but wasn't exactly sure what to put.

Reply:
Could you change this to have a default mock mockBeefyVault with these empty values? Then, in cases like `mockBeefyVault1` and `mockBeefyVault2`, destructure that object and add the appropriate new values. With all the values in each object hardcoded, it's hard for me to tell what's changed between them at a glance.